tests/x11/multi_users_dm.pm

```diff
@@ -75,6 +75,7 @@ sub run {
     # Make sure screen changed before calling handle_login function (for slow workers)
     wait_still_screen;
     handle_login($user, 1);
+    send_key 'esc'                        if match_has_tag('gnome-activities');
```

Comment:
It's a bit weird to have `match_has_tag` without a preceding `assert_screen`, but using `check_screen` would waste some time doing an unnecessary match again...

Reply:
actually curious why this even works - the last assert_screen in handle_login is for displaymanager-password-prompt

Reply:
I put it back in wip, I see what @DimStar77 says and in the VR are run with a different codebase. Sorry for the mess :/